SConstruct

```diff
@@ -429,12 +436,19 @@ consolidateObjectTable = command("consolidateObjectTable", [transformObjectCatal
 
 
 gen3repo = env.Command([os.path.join(REPO, "butler.yaml"), os.path.join(REPO, "gen3.sqlite3")],
-                       [forcedPhotCcd, consolidateObjectTable],
+                       [forcedPhotCcd, forcedPhotCoadd, consolidateObjectTable],
```

Comment:
Nitpick: these two commits became non-orthogonal here. If it's easy for you to either move this change to the previous commit or just squash the whole ticket into one commit, go for it, but not important.

Reply:
I actually wanted two commits so when we have pyarrow on mac os jenkins, we could basically revert everything in the second commit. `forcedPhotCoadd` won't need to be here if we always run the post-processing tasks.